docs/content/CONFIGURATION.md

```diff
@@ -15,18 +15,33 @@ The `.env` file contains a list of all available variables and comments explaini
 The `veda.config.js` file is an additional configuration file for veda.  
 It is through this file that you specify how the Veda content can be found.
 
-This is done by providing a glob path for each one of the [content types](./CONTENT.md). (Datasets, Discoveries).  
+This is done by providing a glob path for each one of the [content types](./CONTENT.md). (Datasets, Stories).  
 The default configuration is:
 ```js
 datasets: './datasets/*.data.mdx'
-discoveries: './discoveries/*.discoveries.mdx'
+stories: './stories/*.stories.mdx'
 ```
 
 ### Page overrides
 To adapt the Veda dashboard to the individual needs of you instance, some content/component overrides are provided. These overrides allow you to alter certain parts of the application, or inject code without having to fork the UI part of veda.
 
 See [PAGE_OVERRIDES](./PAGE_OVERRIDES.md) for a full list of elements to customize.
 
+### Strings
+
+The `strings` object allows you to customize the nomenclature used in some parts of the application so that it better reflects your use case. Since these values will need to be used in different contexts, a plural, singular and zero form of each term is required.
```

Comment:
Should we always enforce that? My hunch is that if we end up using the strings feature a lot, enforcing the zero/plural/singular rule is going to be cumbersome and not needed in the majority of cases.

Reply:
I agree that this adds some overhead. Would your suggestion be to also allow a single string that is used for all values?

Reply:
What would be ideal, possibly, would be to follow the standard used across [i18n projects](https://www.i18next.com/translation-function/plurals), or some form of it:

```
strings: {
  stories_one: 'Story',
  stories_other: 'Stories',
  sheep: 'Sheep' // Sheep's plural is 'Sheep' so don't bother
}
```

So when looking up for a key, first look for a version with suffix (`one`, `other`, `zero`), then fallback on a version without suffix. The zero case fallbacks on `other` by default, but can also be specified as `stories_zero`.

Reply:
We could also do some variation on it if it feels more comfortable:

```
strings: {
  stories: {
    one: 'Story',
    other: 'Stories'
  }
  sheep: 'Sheep' // Sheep's plural is 'Sheep' so don't bother
}
```

Reply:
This is very interesting, and I like the automated version. Not sure it would work for multiple words, like in this case where in the ghg case, "stories" becomes "data stories"

It indeed seems that in English zero is always plural (aside from uncountable), so I agree to go with a more standard approach of `one|other`. I've a slight preference for the `object` example. Unless you have other arguments here.

Reply:
Sounds good, and let's hope we never have to make an [arabic or welsh](https://www.linkedin.com/pulse/internationalization-localization-tips-plurals-daniel-neumann/) version of VEDA 🫠